qemu/tests/virtio_fs_share_data.py

```diff
@@ -75,6 +75,23 @@ def get_stdev(file):
         logging.info("%s device id is %s.", file, stdev)
         return stdev
 
+    def check_socket_group():
+        """
+        check socket path's user group
+        """
+        cmd_get_sock = params["cmd_get_sock"]
+        for device in vm.devices:
+            if isinstance(device, qdevices.QVirtioFSDev):
+                sock_path = device.get_param("sock_path")
+                break
+        logging.info(process.system_output(cmd_get_sock % sock_path))
+        sock_path_info = process.system_output(cmd_get_sock % sock_path)
```

Comment:
Why does execute `process.system_output(cmd_get_sock % sock_path) ` twice?

Reply:
Deleted it. I added it as debug info before.
Actually there is no need to keep.